core-redirects-studio-lib/src/main/java/com/tallence/core/redirects/studio/repository/RedirectRepositoryImpl.java

```diff
@@ -252,6 +255,10 @@ private Redirect convertToRedirect(Content redirectEntry) {
     );
   }
 
+  private Redirect convertToRedirectRef(Content redirectEntry) {
+    return new RedirectImpl(String.valueOf(IdHelper.parseContentId(redirectEntry.getId())));
```

Comment:
Ich würde hier lieber `Integer.toString` benutzen. Zum einen ruft `String.valueOf` eh nix anderes auf, zum anderen muss man dann nicht raten/wissen, welchen Typ `parseContentId` zurück gibt.

```suggestion
    return new RedirectImpl(Integer.toString(IdHelper.parseContentId(redirectEntry.getId())));
```